rewrite-yaml/src/main/java/org/openrewrite/yaml/format/IndentsVisitor.java

```diff
@@ -95,12 +99,24 @@ public IndentsVisitor(IndentsStyle style, @Nullable Tree stopAfter) {
                 y = y.withPrefix(indentComments(y.getPrefix(), indent));
             }
         }
+
+        if (y instanceof Yaml.Scalar && y.getMarkers().findFirst(MultilineScalarChanged.class).isPresent()) {
```

Comment:
If there's no overlap with the above conditionals then it might be clearer to use `else if` here. Any thoughts as to that?
```suggestion
        } else if (y instanceof Yaml.Scalar && y.getMarkers().findFirst(MultilineScalarChanged.class).isPresent()) {
```

Reply:
I agree. When I started writing this, there was a lot of removal / adding of code, until I got it right. In the end it could be an `else-if` after all.